examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,46 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+```html
```

Comment:
This snippet is a good start; however, it is desirable that there is an example page a consumer of the Pubfood repo can load either via local web server and/or filesystem.

In principle, what we see in the _audience-network-bid-delegate.js_ module "should" :tm: work, but generally good to prove that out with a simple integration example page even if Audience Network does not bid.

* On this test Audience Network bids, must a consumer of the repo have their own Audience Network account and placementId or does FB provide a test `{accountId}_{placementId}` for peoples to use?

Like in other header bidding framework adapter modules, it appears that a consumer not only needs an Audience Network account and placementId, there needs to be some additional flag set on the Audience Network system for the Account/Placement?

* `https://an.facebook.com/v2/placementbid.json?sdk=5.5.web&placementids[]=1798405500476623_1798416553808851&adformats[]=300x250`
* `errors:["Application not authorised for header bidding"]`

Could I ask:
1. Create an integration test page
* Perhaps you can base the integration on something from here [Audience Network examples](https://github.com/fbsamples/audience-network/tree/master/samples/mobile_web)

2. Look into a public sandbox/demo `placementId` for use in the example
* The "5...'s" placementId is fine for the commit, but could you enable the placementId above for header bidding if FB do not have a sandbox/demo identifier to use

The primary object with the example integration page is to ensure the respective adapter implementation example bids will indeed flow through the framework and to the AuctionProvider delegate.